zulipterminal/config/keys.py

```diff
@@ -172,6 +171,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Narrow to compose box message recipient',
         'key_category': 'msg_compose',
     },
+    'EXIT_COMPOSE': {
+        'keys': ['esc'],
+        'help_text': 'Stop composing a message and close text box',
+        'key_category': 'msg_compose',
+    },
```

Comment:
NIT: Would it be better to keep this simple and just write it as `Exit message compose box`?